### DISCUSSION TITLE
GitHub Lab is not continuing, I would like to run it as an OSS

### DISCUSSION THREAD

**Discussion by wildcard** [community/community#19875] [GitHub Education]
Please GitHub can you share GitHub lab code as open source so we can host it publicly for everyone that wants to still run the platform

See original https://github.community/t/github-lab-is-not-continuing-i-would-like-to-run-it-as-an-oss/258841?u=wildcard

**Comment by baysideengineer** [comment]
Hey @wildcard, it is always sad when a great product gets shut down. But maybe you can try out [GitHub Skills](https://skills.github.com). It is really great and you can build your own courses. Maybe you can rebuild some courses from [GitHub Learning Lab](https://lab.github.com). Would love to see this. All the best.

**Reply by wildcard** [reply]
GitHub Skills format is great, we're already adopting it, but the **GitHub Lab YAML structure is still superior** to the markdown with sections format.

Open-sourcing GitHub Lab YAML Parser and GitHub Lab Frontend **without the actions/event implementation** would give many of us a good starting point.

This should be the spirit of GitHub, Developer Education is a growing field - it need help from the Open source Community

**Comment by samus-aran** [comment]
Hey @wildcard 👋 

Really appreciate your passion for this topic, but I must lock this and deem it as a duplicate to your [original discussion](https://github.com/community/community/discussions/19875) as it contains the answer best fitting. I'm pretty sure there was a migration gap here from the original discourse board to cause this jump in IDs. 😵‍💫 .
